.azuredevops/build-ci.yml

```diff
@@ -12,49 +12,77 @@ pr:
 variables:
   skipComponentGovernanceDetection: true
 
-pool: 'Lite'
+# The `resources` specify the location and version of the 1ES PT.
+resources:
+  repositories:
+    - repository: m365Pipelines
+      type: git
+      name: 1ESPipelineTemplates/M365GPT
+      ref: refs/tags/release
+extends:
+  # The pipeline extends the 1ES PT which will inject different SDL and compliance tasks.
+  # Read more: https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/1es-pipeline-templates/onboarding/overview
+  template: v1/M365.Official.PipelineTemplate.yml@m365Pipelines
+  parameters:
+    pool:
+      name: Small-1ES # This is one of the Fluid Orgs new 1ES hosted pools.
```

Comment:
@rohandubal this one [still needs permissions on the Small-1ES pool](https://dev.azure.com/fluidframework/public/_build/results?buildId=228714&view=logs&s=9098e30b-84bb-596b-120e-2aa9e89b66cf), that's why the check is stuck.